src/applications/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
```

Comment:
If we want to use `withRouter` you should use the one in the utils folder as it's compatible with react-router v6. There will be a few changes such as using `this.props.navigate()` instead of `this.props.router.push()`, but it will keep things consistent with the rest of the `claims-status` code

Reply:
Thanks for the feedback! I've updated the code.